specification/README.md

```diff
@@ -0,0 +1,21 @@
+# Semantic Conventions
+
+**Status**: [Mixed][DocumentStatus]
```

Comment:
You can mark this as stable.  There's no normative language here to stabilize anyway, but also there are not multiple sections of Stable + Experimental so it cannot be mixed.

Reply:
Wouldn't saying it's `stable` here give folks a false idea that whatever is listed here is stable? What about just not having it?

Reply:
> You can mark this as stable.

Wouldn't that be confusing for readers? Since it's the entry-point couldn't that be misinterpreted as `all of the semantic conventions are stable`?

General question: Is the status required on all README files?

Reply:
I'll remove it for now.